### ISSUE TITLE
paredit.clj reliant on unavailable version of parsley

### ISSUE THREAD

**Issue by laurentpetit** [laurentpetit/ccw#456]
Original [issue 415](https://code.google.com/p/counterclockwise/issues/detail?id=415) created by laurentpetit on 2012-08-30T18:09:05.000Z:

With a new clone of the paredit.clj project, when I try and build (via `mvn clean install`) I get an error trying to resolve the dependency: org.lpetit:net.cgrand.parsley:jar:0.0.9.STABLE001

Trying to build parsley locally has proven futile (&quot;Non-resolvable parent POM: Could not transfer artifact ccw:ccw_pomtemplate:pom:1.0.0-SNAPSHOT ...&quot;).

Long story short, How can I get 0.0.9.STABLE001 of your fork of parsley? Can this be added to clojars?

Your comment on this was: &quot;paredit's pom.xml isn't usable currently&quot;

Thanks for your help,
Tom
